gdpr/impl.go

```diff
@@ -19,6 +19,14 @@ import (
 //
 // Nothing in this file is exported. Public APIs can be found in gdpr.go
 
+type Signal int
+
+const (
+	AmbiguousGDPR Signal = -1
+	NoGDPR        Signal = 0
+	YesGDPR       Signal = 1
+)
```

Comment:
No need to repeat the GDPR name here since it's already package scoped. How about something like:
```
const (
	SignalAmbiguous Signal = -1
	SignalNo        Signal = 0
	SignalYes       Signal = 1
)
```